src/config.ml

```diff
@@ -890,6 +919,22 @@ let a_proto_force =
   *> choice [ string "tcp" *> return `Tcp; string "udp" *> return `Udp ]
   >>| fun prot -> `Proto_force prot
 
+let a_protocol_flags =
+  string "protocol-flags" *> a_whitespace *> commit
+  *> sep_by a_whitespace a_single_param
```

Comment:
Here we can parse an empty `protocol-flags` (which is fine but probably not used much). However, the empty `protocol-flags` requires at least one space or comment at the end to parse successfully. This is maybe not a big deal, but since the server may send such a protocol-flags it's unfortunate to error on that.

I'm not sure how to fix that.

Reply:
Figured it out. Tested manually with Miragevpn.Config.{parse,pp} in a top level.